example/lib/widgets/get_disabled_purpose_ids.dart

```diff
@@ -22,8 +22,9 @@ class _GetDisabledPurposeIdsState
   Future<String> callDidomiPlugin() async {
     final List<String> result = await DidomiSdk.disabledPurposeIds;
     result.sort(); // Required for UI tests
+    print(result.join(","));
     if (result.isEmpty) {
-      return "Disabled Purposes list is empty.";
+      return "Disabled Purpose list is empty.";
     } else {
       return "Disabled Purposes: ${result.join(",")}.";
```

Comment:
Should we put result.join(",") in a variable?

Reply:
I was planning to remove logs afterward, so I think won't be necessary.